tracer/build/_build/Build.ExplorationTests.cs

```diff
@@ -28,21 +32,56 @@ partial class Build
     [Parameter("Indicates if the Dynamic Instrumentation product should be disabled.")]
     readonly bool DisableDynamicInstrumentationProduct;
 
+    [Parameter("Indicates whether the line probe scenario should run.")]
+    readonly bool LineProbes;
+
+    [Parameter("Indicates whether the profiler should output files for diagnostic the instrumentation.")]
+    readonly bool RejitVerify;
+
+    [Parameter("Indicates if the process should wait for debugger attach.")]
+    readonly bool Attach;
+
+    [Parameter("Indicates whether to overwrite existing exploration test project directory or not.")]
+    readonly bool Overwrite;
```

Comment:
It's a nit but I don't think adding these here is a good option - we'd like to _remove_ most parameters if possible, as they add a lot of confusion regarding what they apply to. e.g. `Attach` doesn't apply in _general_.

I think we probably could/should just read these directly from the env vars in the methods below.

Reply:
I agree that tey add confusion. I'll remove those that are only for use in dev env.

Reply:
Fixed in 6853dece4bb3cc91a684157e03ae7d3a18a3de0a